Maya: custom namespace for references (#4511)
ynput/OpenPype: website/docs/admin_hosts_maya.md

This is useful to fix some specific renderer glitches and advanced hacking of Maya Scene files. `Patch name` is label for patch for easier orientation.
`Patch regex` is regex used to find line in file, after `Patch line` string is inserted. Note that you need to add line ending.

## Load Plugins

> ### Reference Loader > Namespace and Group Name

Reviewer: ```suggestion
### Reference Loader

#### Namespace and Group Name
```

This is useful to fix some specific renderer glitches and advanced hacking of Maya Scene files. `Patch name` is label for patch for easier orientation.
`Patch regex` is regex used to find line in file, after `Patch line` string is inserted. Note that you need to add line ending.

## Load Plugins

### Reference Loader

#### Namespace and Group Name
Here you can create your own custom naming for the reference loader.

The custom naming is split into two parts: namespace and group name. If you don't set the namespace or the group name, an error will occur.
Here's the different variables you can use:

<div class="row markdown">
<div class="col col--5 markdown">

| Token | Description |
|---|---|
|`{asset_name}` | Asset name |
|`{asset_type}` | Asset type |
|`{subset}` | Subset name |
|`{family}` | Subset family |

</div>
</div>

> The namespace field can contain a single group of '#' number tokens to indicate where the namespace's unique index should go. The amount of tokens defines the zero padding of the number, e.g ### turns into 001.

Reviewer 2: ```suggestion
The namespace field can contain a single group of '#' number tokens to indicate where the namespace's unique index should go. The amount of tokens defines the zero padding of the number, for example `###` turns into `001`.
```

- **Compression type** which file encoding to use.
- **Data format** what format is the file encoding.
- **Quality** lets you control the compression value for the output. Results can vary depending on the compression you selected. Quality values can range from 0 to 100, with a default value of 95.
- **Background Color** the viewports background color.
- **Background Bottom** the viewports background bottom color.
- **Background Top** the viewports background top color.
- **Override display options** override the viewports display options to use what is set in the settings.
- **Isolate view** isolates the view to what is in the review instance. If only a camera is present in the review instance, all nodes are displayed in view.
- **Off Screen** records the playblast hidden from the user.
- **2D Pan/Zoom** enables the 2D Pan/Zoom functionality of the camera.
- **Renderer name** which renderer to use for playblasting.
- **Width** width of the output resolution. If this value is `0`, the asset's width is used.
- **Height** height of the output resolution. If this value is `0`, the asset's height is used.

> #### Viewport Options

Reviewer 2: This looks like you accidentally deleted some settings on a merge conflict. I suspect these shouldn't be removed. Right? There some other changes in this file too that I suspect shouldn't be here either.

Author: Oh yeah I just realized I removed some things that I shouldn't have ! I will revert my changes

This is useful to fix some specific renderer glitches and advanced hacking of Maya Scene files. `Patch name` is label for patch for easier orientation.
`Patch regex` is regex used to find line in file, after `Patch line` string is inserted. Note that you need to add line ending.

## Load Plugins

### Reference Loader > Namespace
Here you can create your own custom naming for the reference loader. <br>

> The custom naming is split into two parts separated by a ":" symbol. The first half is the namespace and the second half is the group name of the reference. If you don't set the namespace or the group name (or both), the default namespace and group name will be applied.

Author: So if I understand correctly, you want this field to be filled by default with the originally generated namespace, and not blank ?

> The custom naming is split into two parts separated by a ":" symbol. The first half is the namespace and the second half is the group name of the reference. If you don't set the namespace or the group name (or both), the default namespace and group name will be applied.

Reviewer 2: Yes, the result will be the same but then it could _always_ use the value from settings. Seems more explicit to me. No? I wonder what the OP crew thinks however.

> The custom naming is split into two parts separated by a ":" symbol. The first half is the namespace and the second half is the group name of the reference. If you don't set the namespace or the group name (or both), the default namespace and group name will be applied.

Author: Yes I agree with you, it could even be simpler. Do I wait for feedback from OP crew or do I make the modification now ?

> The custom naming is split into two parts separated by a ":" symbol. The first half is the namespace and the second half is the group name of the reference. If you don't set the namespace or the group name (or both), the default namespace and group name will be applied.

Reviewer 2: @iLLiCiTiT @mkolar any thoughts on this by any chance? (not sure who best to tag, sorry)

![Extract Playblast Settings](assets/maya-admin_extract_playblast_settings_camera_options.png)

> 
> ## Load Plugins
> 
> ### Reference Loader > Namespace
> Here you can create your own custom naming for the reference loader.
> 
> The custom naming is split into two parts separated by a ":" symbol. The first half is the namespace and the second half is the group name of the reference. If you don't set the namespace or the group name, an error will occur.
> Here's the different variables you can use:
> 
> <div class="row markdown">
> <div class="col col--5 markdown">
> 
> | Token | Description |
> |---|---|
> |`{asset_name}` | Asset name |
> |`{asset_type}` | Asset type |
> |`{subset}` | Subset name |
> |`{family}` | Subset family |
> 
> </div>
> </div>
> 
> Example: `{asset_name}_{subset}_:_GRP`
> 

Reviewer: ```suggestion
```
